clients/fides-js/src/components/fides.css

```diff
@@ -262,7 +263,6 @@ div#fides-button-group {
 button.fides-banner-button {
   font-size: var(--fides-overlay-font-size-buttons);
   display: inline-block;
-  flex: auto;
```

Comment:
this was unnecessary and causing the Manage button to grow unexpectedly

Reply:
Note: this did cause a regression which was fixed in https://github.com/ethyca/fides/pull/5161